src/Core/Deprecated/Undeprecator.js

```diff
@@ -74,4 +74,12 @@ export const deprecatedFeature2MeshOptions = (options) => {
     }
 };
 
+export const deprecatedC3DEngineWebGLOptions = (options) => {
+    if (options.isWebGL2 !== undefined) {
+        if (!options.isWebGL2) {
```

Comment:
replace these two if by `options.isWebGL2 === false` (simpler and handles null case)

Reply:
Changed !
I first wanted to deprecate `isWebGL2: true` but we may want to use this parameter later when three will add WebGPU support !